# params.yaml
trajectory_predictor_node:
  ros__parameters:
    position_model_path: /home/user/shared_volume/ros2_ws/src/DronePathPredictor_ros/config/position_mix_best_model.pth
    velocity_model_path: /home/user/shared_volume/ros2_ws/src/DronePathPredictor_ros/config/velocity_mix_best_model.pth
    position_npz_path: /home/user/shared_volume/ros2_ws/src/DronePathPredictor_ros/config/position_mix_norm_stats.npz
    velocity_npz_path: /home/user/shared_volume/ros2_ws/src/DronePathPredictor_ros/config/velocity_mix_norm_stats.npz
    buffer_duration: 2.0
    dt: 0.1
    pos_input_length: 20
    pos_output_length: 10
    pos_hidden_dim: 64
    pos_num_layers: 2

    vel_input_length: 20
    vel_output_length: 10
    vel_hidden_dim: 64
    vel_num_layers: 2

    use_velocity_prediction: false
